tests/memory_tests/CMakeLists.txt

```diff
@@ -0,0 +1,25 @@
+# Copyright (C) 2018-2021 Intel Corporation
+# SPDX-License-Identifier: Apache-2.0
+#
+
+cmake_minimum_required(VERSION 3.13)
+
+set (CMAKE_BUILD_TYPE "Release" CACHE STRING "Choose the build type")
+
+project(memory_tests)
+
+set(OpenVINO_SOURCE_DIR "${CMAKE_CURRENT_SOURCE_DIR}/../../")
+
+# Search OpenVINO Inference Engine installed
+find_package(InferenceEngine)
+
+if(NOT InferenceEngine_FOUND)
```

Comment:
please use [CMakeList.txt](https://github.com/openvinotoolkit/openvino/blob/master/tests/stress_tests/CMakeLists.txt) as example for update current file